src/_guides/language/language-tour.md

```diff
@@ -2217,13 +2216,11 @@ for a class. Constructor names can be either <code><em>ClassName</em></code> or
 
 <?code-excerpt "misc/test/language_tour/classes_test.dart (object-creation)" replace="/ as .*?;/;/g"?>
 {% prettify dart %}
-var jsonData = jsonDecode('{"x":1, "y":2}');
-
 // Create a Point using Point().
 var p1 = new Point(2, 2);
 
 // Create a Point using Point.fromJson().
-var p2 = new Point.fromJson(jsonData);
+var p2 = new Point.fromJson({'x': 1, 'y': 2});
```

Comment:
We had been using `jsonDecode('{"x":1, "y":2}');` but things were getting messy with the types. So I dropped the decoding (these examples are about object creation anyways), and just passed a "JSON literal" as an argument instead.